samples/python/p3m.py

```diff
@@ -115,10 +115,22 @@
     system.part[2 * i + 1].q = 1.0
 # P3M setup after charge assigned
 #############################################################
-p3m = electrostatics.P3M_GPU(bjerrum_length=1.0, accuracy=1e-2)
+
+print("\nSCRIPT--->Create p3m\n")
+p3m = electrostatics.P3M(bjerrum_length=2.0, accuracy=1e-2)
+
+print("\nSCRIPT--->Add actor\n")
 system.actors.add(p3m)
 
-print("P3M parameter:\n")
+print("\nSCRIPT--->P3M parameter:\n")
+p3m_params = p3m.get_params()
+for key in p3m_params.keys():
+    print("{} = {}".format(key, p3m_params[key]))
+
+print("\nSCRIPT--->Explicit tune call\n")
+p3m._tune() 
```

Comment:
Maybe there should be a method of the P3M class to call the tune routine instead of using the as "internal" marked `_tune` method.